### PR TITLE
[ICDS Dashboard] THR data fill

### PR DESCRIPTION
**Pull request by challabeehyv**
https://dimagi-dev.atlassian.net/browse/ICDS-1499

##### SUMMARY
THR data fill

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/management/commands/fix_thr_past_data.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        end_month = dateutil.parser.parse(end_month).date().replace(day=1)
+        with open(path, "r", encoding='utf-8') as sql_file:
+            sql_to_execute = sql_file.read()
+
+        while end_month >= start_month:
+            print("PROCESSING MONTH {}".format(end_month))
+            next_month = end_month + relativedelta(months=1)
+            query = sql_to_execute.format(month=end_month, next_month=next_month)
+            _run_custom_sql_script(query)
+            end_month -= relativedelta(months=1)
```
Comment (Rohit25negi) [inline custom/icds_reports/management/commands/fix_thr_past_data.py:41]: is it to run it for months in reverse direction?

**Review by Rohit25negi — APPROVED** [review]
Looks good. Though left one question

File: custom/icds_reports/management/commands/fix_thr_past_data.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/management/commands/fix_thr_past_data.py:41]: yes

**Review by Rohit25negi — APPROVED** [review]
Looks good. We can run it in limited release.

**Comment by challabeehyv** [comment]
closing this as the pr is raised against new repo: https://github.com/dimagi/commcare-icds/pull/11

**Pull request closed without merging**